tests/functional/test_functional.py

```diff
@@ -0,0 +1,31 @@
+from flask import url_for
+
+
+class TestFunctional(object):
+
+    def test_can_show_email_confirmation(self, live_server, browser):
+        browser.visit(url_for('main.request_email_address', _external=True))
+
+        assert ('GOV.UK - The best place to find government services and '
+                'information') in browser.title
```

Comment:
This was just an initial test to confirm that selenium is working, can be removed as doesn't do anything except go to the page